kokkos/src/bude.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+#include <iomanip>
+
+#ifdef KOKKOS_IN_TREE
+	#include <Kokkos_Core.hpp>
+#else
+	#include <kokkos/Kokkos_Core.hpp>
+#endif
+
+#ifndef DEFAULT_WGSIZE
+#define DEFAULT_WGSIZE 64
```

Comment:
Make the default size 128. This is a good choice on both CXL and Rome.

Reply:
Done